src/irr/asset/IAssetManager.cpp

```diff
@@ -251,21 +234,20 @@ void main()
             gl_Position = irr_glsl_pseudoMul4x4with3x1(irr_builtin_glsl_workaround_AMD_broken_row_major_qualifier_mat4x4(CamData.params.MVP), vPos);
 	        uv = vTexCoord;
         }
-		)===", asset::ISpecializedShader::ESS_VERTEX, "irr/builtin/materials/lambertian/uv_debug_shader/specializedshader");
+		)===", asset::ISpecializedShader::ESS_VERTEX, { "irr/builtin/materials/debug/uv_debug_shader/specializedshader",
```

Comment:
UV debug shader should not be using a texture, it should be outputting the UV coords as color!

Reply:
none of the default shaders PLY uses for its pipelines can use a texture! (because it will never load a texture)

Reply:
hey I fixed it!